sfge/src/main/resources/log4j2.xml

```diff
@@ -3,7 +3,7 @@
 	<!-- status = The level of internal Log4j events that should be logged to the console. -->
     <Appenders>
 		<CliMessagerAppender name="LogToCliMessager">
-			<PatternLayout pattern="%d{yyyy-MM-dd HH:mm:ss} %m%n"/>
+			<PatternLayout pattern="%c{1}:%L %m%n"/>
```

Comment:
The date format is unnecessary padding, but the file name and line number are needed because the only available stack trace stops at `CliMessagerAppender`. This isn't as thorough a stack as I would have wanted, but I believe it to be the best we can do under these circumstances.

Reply:
We still display the time for CliMessager events meant for users on console. If you think the needs are different, should we keep the appenders separate? A new Telemetry appender and the existing CliMessager appender?